repoman/cli.py

```diff
@@ -70,8 +71,10 @@ def cli(ctx, workspace, verbose, remote_base, config):
               help="Checkout package into this directory.")
 @click.option('--develop', is_flag=True,
               help="Ignore tags in name list and check out development branches")
+@click.option('--manifest', is_flag=True,
```

Comment:
`--bom` `Provider a Bill of Materials (bom) that describes the commits checked out`